blocks/form/gdpr-confirmation.js

```diff
@@ -0,0 +1,78 @@
+// get url parameters
+const curUrl = window.location.href;
+const urlParams = new URLSearchParams(window.location.search);
+const paramdata = urlParams.get('data');
+const paraminit = urlParams.get('init');
+
+function hasQueryParams() {
+  return curUrl.toString().includes('?');
+}
+
+// Send a query parameters to the GDPR API Accept
+export async function acceptresult(accept, urlProd, secondConfig) {
```

Comment:
this function does not need to be exported as it is not used by any other file, also please do `camelCase` names for for functions and parameters: `acceptResult`, `paramData`, `paramInit` etc.